javagen/src/main/java/com/azure/autorest/model/clientmodel/ClientBuilderTrait.java

```diff
@@ -205,15 +205,15 @@ private static String addLogging(LogLevel level, String message) {
         } else {
             switch (level) {
                 case VERBOSE:
-                    return String.format("LOGGER.log(LogLevel.VERBOSE, () -> \"%s\");", message);
+                    return String.format("LOGGER.atVerbose().log(() -> \"%s\");", message);
                 case INFORMATIONAL:
-                    return String.format("LOGGER.log(LogLevel.INFORMATIONAL, () -> \"%s\");", message);
+                    return String.format("LOGGER.atInfo().log(() -> \"%s\");", message);
                 case WARNING:
-                    return String.format("LOGGER.log(LogLevel.WARNING, () -> \"%s\");", message);
+                    return String.format("LOGGER.atWarning().log(() -> \"%s\");", message);
                 case ERROR:
-                    return String.format("LOGGER.log(LogLevel.ERROR, () -> \"%s\");", message);
+                    return String.format("LOGGER.atError().log(() -> \"%s\");", message);
                 default:
-                    return String.format("LOGGER.log(LogLevel.INFORMATIONAL, () -> \"%s\");", message);
+                    return String.format("LOGGER.atInfo().log(() -> \"%s\");", message);
```

Comment:
Will this change the logging output style or just change the way we write logs?

Reply:
This part is unbranded.

Though for branded, we are recommended to output this pattern as well. https://github.com/Azure/autorest.java/issues/2631

Reply:
> Will this change the logging output style or just change the way we write logs?

Both - but this change is only for unbranded. For branded, we should use this pattern too but we can do that in a separate PR after we review all the current logging calls we generate today.